src/pages/paper-form.tsx

```diff
@@ -1,7 +1,18 @@
+import { useAppCtx } from '@store';
+import { Theme } from '@types';
 import { NextPage } from 'next';
+import { useRouter } from 'next/router';
 import React from 'react';
+import { useEffect } from 'react';
 
 const PaperForm: NextPage = () => {
+  const Router = useRouter();
+  const { state } = useAppCtx()
+  useEffect(() => {
```

Comment:
is this just to redirect us back to the modern form if the theme is something other than astrophysics?

I wonder if we should maybe move this logic upward into `_app` and possible into a hook that does this check, hmm

Reply:
Right. If current path is classic or paper form, but theme is not astrophysics. I'll look into this, let me know if you already have some ideas in mind.

Reply:
Moved the redirect logic to `_app`